config/config.go

```diff
@@ -9,17 +9,21 @@ import (
 
 	"github.com/BurntSushi/toml"
 	"github.com/ardanlabs/conf"
+	"github.com/joho/godotenv"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
 
 const (
 	DefaultConfigPath = "config/config.toml"
+	DefaultEnvPath    = "config/.env"
 	ConfigFileName    = "config.toml"
 	ServiceName       = "ssi-service"
 	ConfigExtension   = ".toml"
 
-	DefaultServiceEndpoint = "http://localhost:8080"
+	DefaultServiceEndpoint      = "http://localhost:8080"
+	EnvVariableKeystorePassword = "KEYSTORE_PASSWORD"
```

Comment:
may want to make a `type EnvironmentVariable string` for these

Reply:
Yea that makes sense